crates/bevy_dylib/src/lib.rs

```diff
@@ -1,12 +1,26 @@
+#![warn(missing_docs)]
 #![allow(clippy::single_component_path_imports)]
 
 //! Forces dynamic linking of Bevy.
 //!
 //! Dynamically linking Bevy makes the "link" step much faster. This can be achieved by adding
-//! `bevy_dylib` as dependency and `#[allow(unused_imports)] use bevy_dylib` to `main.rs`. It is
-//! recommended to disable the `bevy_dylib` dependency in release mode by adding
-//! `#[cfg(debug_assertions)]` to the `use` statement. Otherwise you will have to ship `libstd.so`
-//! and `libbevy_dylib.so` with your game.
+//! `bevy_dylib` as a dependency and adding the following code to the `main.rs` file:
```

Comment:
You can now simple enable the `dynamic` feature of the `bevy` crate instead. I think this should be recommended over directly including bevy_dylib.

Reply:
Agreed. I added all three ways of enabling the dynamic linking of bevy.
1. Using the `--features bevy/dynamic` flag.
1. Using the `Cargo.toml` features.
1. Using the `bevy_dylib` dependency.